applications/callflow/src/module/cf_resources.erl

```diff
@@ -92,6 +92,7 @@ build_offnet_request(Data, Call) ->
                             ,{<<"Bypass-E164">>, get_bypass_e164(Data)}
                             ,{<<"Inception">>, get_inception(Call)}
                             ,{<<"B-Leg-Events">>, [<<"DTMF">>]}
+			    ,{<<"Custom-Channel-Vars">>, {[{<<"Authorizing-ID">>, whapps_call:authorizing_id(Call)}]}}
```

Comment:
move this to get_channel_vars as we may want to set more vars in the future.

`{<<"Custom-Channel-Vars">>, get_channel_vars(Call) }`

```
get_channel_vars(Call) ->
   wh_json:from_list(
     [{<<"Authorizing-ID">>, whapps_call:authorizing_id(Call)}]
   ).
```